weave-js/src/components/UserLink.tsx

```diff
@@ -233,7 +232,7 @@ type UserLinkProps = {
   hasPopover?: boolean; // Can you click to open more a popup
 };
 
-const fetchUser = (userId: string) => {
+const fetchUser = (userId: string, apolloClient: ApolloClient<object>) => {
```

Comment:
Is the concrete type known here so we can avoid `object`?

Reply:
nope (: it is just the cache shape which we don't care about - `any` would also suffice

Reply:
I think you might be able to omit the type parameter then? 🤔